### PR TITLE
fixed indentation

### PR DESCRIPTION
**Pull request by 81jumpstr**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-			endwhile;
-			endif;
-		?>
-	  </ul>
-	</div>
+  <div class='main-section'>
+    <?php get_sidebar(); ?>
+    <ul class="posts">
+      <?php 
+        if ( have_posts() ) :
```
Comment (royscheeren) [inline index.php]: Quick philosophical question: what would your markup look like in the off chance there are 0 posts?

**Review by royscheeren — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (81jumpstr) [reply index.php]: I imagine; the background image, a <header> and the <footer> only...

**Comment by 81jumpstr** [comment]
I replied on github...

**Comment by 81jumpstr** [comment]
Ha... this is weird. Ok, I replied to your question in "view changes"

File: index.php (hunk 1 above)
Comment (royscheeren) [reply index.php]: Take a pen and paper and go through the steps, think about what the code will write when there are 0 posts.

Comment (81jumpstr) [reply index.php]: No post, no content... So, nothing?
- background image, sidebar, header the footer

**Comment by 81jumpstr** [comment]
Syntax correction for ul

File: index.php [hunk 2]
```diff
@@ -2,19 +2,23 @@
...
+      if ( have_posts() ) :
+    ?>
+        <ul class="posts">
+      <?php
+        while ( have_posts() ) : the_post();
       ?>
-      <li>
-        <?php the_content('&raquo;'); ?>
-      </li>
+        <li>
```
Comment (royscheeren) [inline index.php]: Almost there, now its just your indentation. the ul > while > li. Now it's all on the same level.

**Review by royscheeren — CHANGES_REQUESTED** [review]
Almost there

**Comment by 81jumpstr** [comment]
Ah, yes... I see.

**Review by royscheeren — APPROVED** [review]

**Pull request merged**